installers/pkg.yaml

```diff
@@ -66,6 +69,9 @@ dependencies:
     platform: linux/arm64
     from: /dtb/rockchip/rk3399-rock-4se.dtb
     to: /rootfs/artifacts/arm64/dtb/rockchip/rk3399-rock-4se.dtb
+  - image: ghcr.io/nberlee/dtb-talos-turing-rk1:db62b12
```

Comment:
I guess https://github.com/nberlee/dtb-talos-turing-rk1/blob/release-1.9/kernel/build/pkg.yaml#L18 this can also be done as a separate stage using toolchain (since it should have dtc) and just copy over

Reply:
Yes, if you are fine with that, I am happy to do that.

Reply:
yeh, let's do that, also less stuff to maaintain